src/Core/Translation/Locale/Converter.php

```diff
@@ -44,7 +44,7 @@ final class Converter
     /**
      * @param string $translationsMappingFile
      */
-    public function __construct($translationsMappingFile)
+    public function __construct(string $translationsMappingFile)
```

Comment:
```suggestion
    public function __construct(string $translationsMappingFile): void
```

Reply:
Do we need to specify return type for all constructors ?

Reply:
I dont think it's necessary 🤔  . What value does it bring?

Reply:
oh my bad about the return type here, I was tired :sweat: